packages/function-entry/src/function-entry.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import debug from 'debug';
+import { withStyles } from 'material-ui/styles';
+import Input from './input';
+
+const log = debug('pie-elements:text-entry');
+
+const FunctionEntryStyles = {
```

Comment:
Sorry should have said - we can prob skip this for now - theming is something we're going to get to later.